Functions/xASL_quant_Basil.m

```diff
@@ -1,7 +1,7 @@
-function [CBF_nocalib, resultFSL] = xASL_quant_Basil(PWI, x)
+function [CBF_nocalib, ATT_map, resultFSL] = xASL_quant_Basil(PWI, x)
```

Comment:
Not sure what you talk about. `isvar` is not used in the function. We only check existence of ATT_map output from BASIL. That cannot be initialized, but has to be checked.

Reply:
Couldn't find `isvar` either.

Reply:
What I mean, is that if there is no output from BASIL (e.g., with singlePLD), you will use `NaN` for ATT_map. To avoid the need of `if ~exist(ATT_map,'var');ATT_map = NaN; end` at the end, you can initialize `ATT_map` with `NaN` at the start. This is useful as singlePLD should still be the default (unless multiPLD is specifically designated).

Reply:
Currently we would need something like `isvar` (probably `exist(,'var')` instead) at the end. I want to avoid this. But it wasn't there, I was just giving this as example. If by whatever if statements that are skipped, the function end is reached, we need to catch `ATT_map` not existing as variable. Better (cleaner, easier to read and debug) to initialize this at the beginning.

Reply:
As previously discussed - we keep this as it is. Can you resolve?!